app/src/db/migrations/versions/2024_09_03_add_num_splits_and_split_index.py

```diff
@@ -0,0 +1,70 @@
+"""Add num_splits and split_index
+
+Revision ID: 9543d555fad7
+Revises: 39d60b8215fa
+Create Date: 2024-09-03 15:05:51.502359
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "9543d555fad7"
+down_revision = "39d60b8215fa"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("chunk", sa.Column("num_splits", sa.Integer(), nullable=False))
+    op.add_column("chunk", sa.Column("split_index", sa.Integer(), nullable=False))
+    op.alter_column(
+        "document",
+        "dataset",
+        existing_type=sa.TEXT(),
+        nullable=False,
+        comment=None,
```

Comment:
Is this removing the DB comment? If so, why?

Reply:
Good catch, it is. It's removing them because the comments weren't attached in the code definition in document.py. I'll add them in as part of this PR.